core/buildTables.ml

```diff
@@ -52,7 +53,7 @@ struct
         let rec bind o =
           function
             | [] -> o
-            | (`Let (x, _))::bs ->
+            | (Ir.Let (x, _))::bs ->
```

Comment:
That constructor is prefixed with `Ir.` but other in this file are not. Accidental or on purpose?

Reply:
good catch!